### PR TITLE
mrc-3068 upgrade to Java 11

### PR DESCRIPTION
**Pull request by hillalex**
Upgrade to Java 11. A few minor changes to the docker images, update to datetime tests to account for the fact that datetime precision coming out of the db is now only to milliseconds, and a change to the hoverfly code as apparently `keep-alive` header is no longer being set. Is the latter a problem?! Manual testing suggests not, the login flow works as expected when I run it locally. I have deployed this to UAT but can't test it because vpn access not working :roll_eyes:

### CODE REVIEW

File: buildkite/common [hunk 1]
```diff
@@ -1,13 +1,13 @@
 #!/usr/bin/env bash
 set -ex
 
-if [ "$BUILDKITE" = "true" ]; then
```
Comment (hillalex) [inline buildkite/common:4]: this doesn't work locally

File: src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/database_tests/ReportRepositoryTests/ReportTests.kt [hunk 2]
```diff
@@ -334,7 +334,7 @@ class ReportTests : CleanDatabaseTests()
     @Test
     fun `can get latest dates for selected reports`()
     {
-        val now = Instant.now()
+        val now = Instant.ofEpochMilli(1655378424228)
```
Comment (hillalex) [inline src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/database_tests/ReportRepositoryTests/ReportTests.kt:337]: `Instant.now()` gives greater precision that the datetimes coming out of the db, so create an Instant that has millisecond precision.

**Review by EmmaLRussell — COMMENTED** [review]
Can confirm UAT is looking good. 

Also looks like it's fine working locally, though I did have to change `cli.sh` to target this branch when running the cli container. 

> apparently keep-alive header is no longer being set. Is the latter a problem?!

Weird. I guess we'll find out!

**Review by LekanAnni — APPROVED** [review]
LGTM 👍

File: Dockerfile [hunk 3]
```diff
@@ -1,11 +1,30 @@
-FROM vimc/node-docker:master
+FROM openjdk:11
```
[the remark this answers is not in the document]
Comment (LekanAnni) [reply Dockerfile:1]: I think you can add a ticket for updating the docker image.

**Review by EmmaLRussell — APPROVED** [review]

**Pull request merged**